server/routerlicious/packages/lambdas/src/deli/checkpointContext.ts

```diff
@@ -74,7 +85,8 @@ export class CheckpointContext {
         if (checkpoint.clear) {
             updateP = this.checkpointManager.deleteCheckpoint(checkpoint);
         } else {
-            const deliCheckpoint: IDeliState = { ...checkpoint };
+            // clone the checkpoint
+            const deliCheckpoint: IDeliState = { ...checkpoint.deliState };
```

Comment:
Just to double check, this does not change the checkpoint schema in any shape or form right? We still checkpoint the same thing that we used to so backward and forward compatibility should not get impacted?

Reply:
Correct. Before it was including the IQueuedMessage object in the checkpoint under a `queuedMessage` property. That property does not exist in `IDeliState`, so it was an extra property / unused data